docs/quickstart.ipynb

```diff
@@ -34,14 +34,14 @@
   },
```

Comment:
<p>I think we can also add one more example of Wald test to make the user understand how to construct <code>R</code> matrix and <code>q</code>  vector like the following :</p><p> </p><p>You can also test the complicated joint null hypothesis of \X_1 + 2*X2 = 2.0\ and \X_2 = 1.0 \  by setting <code>R</code> and <code>q</code> as the following </p><p> </p>

<pre class="ql-syntax" spellcheck="false">R1 =np.array([[1,2], [0,1]])
q1 = np.array([2.0, 1.0])
fit.wald_test(R = R1, q = q1)
</pre><p> </p><p>How do you think?</p><p>	    </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/py-econometrics/pyfixest/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>

Reply:
Solved with [a86f14d](https://github.com/py-econometrics/pyfixest/pull/569/commits/a86f14d98c426c45b09c901f66ec07dd8c7b36d1) =)

Reply:
Added with [fa5cbec](https://github.com/py-econometrics/pyfixest/pull/569/commits/fa5cbec20c7ab1e263a70b63ce6eab25d622d9b8)